joint_state_controller/test/test_joint_state_controller.cpp

```diff
@@ -251,10 +268,25 @@ TEST_F(JointStateControllerTest, DynamicJointStatePublishTest)
   rclcpp::MessageInfo msg_info;
   ASSERT_TRUE(subscription->take(dynamic_joint_state_msg, msg_info));
 
-  // checking positions is enough
   const size_t NUM_JOINTS = 3;
-  ASSERT_EQ(dynamic_joint_state_msg.joint_names.size(), NUM_JOINTS);
-  ASSERT_EQ(dynamic_joint_state_msg.interface_values[0].values[0], 1.1);
-  ASSERT_EQ(dynamic_joint_state_msg.interface_values[1].values[0], 2.1);
-  ASSERT_EQ(dynamic_joint_state_msg.interface_values[2].values[0], 3.1);
+  const auto INTERFACE_NAMES = {"position", "velocity", "effort"};
```

Comment:
```suggestion
  const auto INTERFACE_NAMES = {hardware_interface::HW_IF_POSITION, 
                                                           hardware_interface::HW_IF_VELOCITY, 
                                                           hardware_interface::HW_IF_EFFORT};
```

Reply:
yup

Reply:
there was another one of these, done that too